app/components/GoogleAnalytics.tsx

```
@@ -0,0 +1,24 @@
+import Script from "next/script";
+
+function GoogleAnalytics({ ga_id }: { ga_id: string }) {
+  return <>
+    <Script
+      async
+      src={`https://www.googletagmanager.com/gtag/js? 
+      id=${ga_id}`}
+     />
+    <Script
+      id="google-analytics"
+      dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)